tests/test_a11y.py

```diff
@@ -176,6 +184,35 @@ def test_axe_core(
 
     # Check found violations against known violations that we do not plan to fix
     filtered_violations = filter_ignored_violations(results["violations"], url_pathname)
+
+    # We expect notebook outputs on the PyData Library Styles page to have color
+    # contrast failures.
+    if url_pathname == "/examples/pydata.html":
+        # All violations should be color contrast violations
+        for violation in filtered_violations:
+            assert (
+                violation["id"] == "color-contrast"
+            ), f"Found {violation['id']} violation (expected color-contrast): {format_violations([violation])}"
+
+        # Now check that when we exclude notebook outputs, the page has no violations
+
+        results_sans_nbout = page.evaluate(
+            f"axe.run({{ include: '{selector}', exclude: '.nboutput > .output_area' }})"
+        )
+        violations_sans_nbout = filter_ignored_violations(
+            results_sans_nbout["violations"], url_pathname
+        )
+
+        # No violations on page when excluding notebook outputs
+        assert len(violations_sans_nbout) == 0, format_violations(violations_sans_nbout)
+
+        # TODO: for color contrast issues with common notebook outputs
+        # (ipywidget tabbed panels, Xarray, etc.), should we override
+        # third-party CSS with our own CSS or/and work with NbSphinx, MyST-NB,
+        # ipywidgets, and other third parties to use higher contrast colors in
+        # their CSS?
```

Comment:
cc @trallard

Reply:
Larger conversation but longer term it would be best to have these fixed upstream vs in here. 
Not sure how much of an effort this would be so will have to pin this for now.

Reply:
Agreed